<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelezionaFormaVolto</class>
 <widget class="QDialog" name="SelezionaFormaVolto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonOvale">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>140</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRotondo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>140</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuadrato">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>140</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDiamante">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>140</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCuore">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>140</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRettangolare">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>140</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTriangolare">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>140</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
